airflow/providers/postgres/hooks/postgres.py

```diff
@@ -95,7 +96,7 @@ def get_conn(self):
         self.conn = psycopg2.connect(**conn_args)
         return self.conn
 
-    def copy_expert(self, sql, filename, open=open):
+    def copy_expert(self, sql, filename, open=open):  # pylint: disable=redefined-builtin
```

Comment:
Rename or #TODO

Reply:
remove it, due to only use in unittest

Reply:
WDYT

Reply:
I am ok with that but then we have to add note in UPDATING.md describing what has changed as this is public interface (someone can use this hook in custom operators).